app/database/models.py

```diff
@@ -431,6 +431,15 @@ class InternationalDays(Base):
     international_day = Column(String, nullable=False)
 
 
+class Location(Base):
+    __tablename__ = "locations"
+
+    id = Column(Integer, primary_key=True, index=True)
+    country = Column(String, nullable=False)
+    city = Column(String, nullable=False)
+    zip_number = Column(String, nullable=False)
```

Comment:
Are there really no python modules that handle this? What about https://pypi.org/project/geopy/? I wouldn't reinvent the wheel here...